### DISCUSSION TITLE
Links to README.md No Longer Work

### DISCUSSION THREAD

**Discussion by EvolvingSysadmin** [GitbookIO/community#814] [0. General bugs and questions]
Links on my GitBook site that link to my README.md file no longer work properly. Wherever there's a link to README.md (my site's homepage) it just points to whatever page I'm currently on, for some reason. This happens with links in the navigation, links within my content, and the link from the site logo image. This problem just started, and I noticed that it's happening on other GitBook sites as well.

**Comment by addisonschultz** [comment]
Hi @EvolvingSysadmin - can you link your site so we can investigate?

**Reply by aikoii** [reply]
@EvolvingSysadmin We've released a fix for this! 🎉 You may just need to hard-refresh your browser to see those changes taking effect.
